spec/integration/deploy_to_heroku_spec.cr

```diff
@@ -18,7 +18,9 @@ require "../spec_helper"
         should_run_successfully "heroku buildpacks:add https://github.com/heroku/heroku-buildpack-nodejs"
       }
 
-      HTTP::Client.get("https://#{app}.herokuapp.com").body.should contain("Welcome to Lucky")
+      # For some reason using HTTP::Client raises an End of file error
+      # So instead we use `curl` to circumvent the issue
+      `curl https://#{app}.herokuapp.com`.should contain("Welcome to Lucky")
```

Comment:
Figured it out @edwardloveall! (well, sort of).

It appears that for some reason the Crystal HTTP::Client raises, but curl (and opening the url in the browser) both work. So to get this working I switched to `curl` for now.